app/main.py

```diff
@@ -104,3 +105,6 @@ async def lifespan(app: FastAPI, registry: svcs.Registry):
     tags=["Portal Settings"],
 )
 app.include_router(auth.router, prefix="/auth", tags=["Auth"])
+
```

Comment:
If I don't remember wrong, we removed this to avoid sonar complain about it. you can easily run it using the included fastapi cli. See [https://fastapi.tiangolo.com/fastapi-cli/#fastapi-dev](https://fastapi.tiangolo.com/fastapi-cli/#fastapi-dev)